File: app/src/main/java/com/young/common/inject/Inject.java
package com.young.common.inject;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Author: landerlyoung
 * Date:   2014-10-21
 * Time:   10:39
 * Life with passion. Code with creativity!
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.FIELD)
public @interface Inject {
    int value();
}

File: app/src/main/java/com/young/common/inject/Inject.kt
package com.young.common.inject

/**
 * Author: landerlyang
 * Date:   2014-10-21
 * Time:   10:39
 * Life with passion. Code with creativity!
 */
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.FIELD)
annotation class Inject(val value: Int)
